File: 2022/day01.py
with open("input_files/day01", "r") as f:
    data = f.read().split("\n\n")
    data = [[int(item) for item in pack.splitlines()] for pack in data]
    data = [sum(pack) for pack in data]
    data = sorted(data)

print(data[-1])
print(sum(data[-3:]))

File: koltin/src/main/kotlin/y2022/day01.kt
package y2022

import read

fun main() {
    val data = read("y2022/day01")
        .split("\n\n")
        .map { it.lines() }
        .map { pack -> pack.map { it.toInt() } }
        .map { it.sum() }
        .sortedDescending()

    println(data[0])
    println(data.take(3).sum())
}
